packages/yoshi-flow-editor-runtime/src/viewerScript.ts

```diff
@@ -193,11 +213,42 @@ const initializeExperiments = () => {
   );
 };
 
-export const initAppForPageWrapper = (initAppForPage: Function) => (
-  ...args: Array<any>
+export const initAppForPageWrapper = (
+  initAppForPage: Function,
+  sentry: SentryConfig | null,
+) => (
+  initParams: IAppData,
+  apis: IPlatformAPI,
+  namespaces: IWixAPI,
+  platformServices: IPlatformServices,
+  ...other: Array<any>
 ) => {
+  if (sentry) {
```

Comment:
mb `sentry && enabled`?
```suggestion
  if (sentry && (process.env.FORCE_SENTRY || process.env.NODE_ENV === 'production')) {
```

Reply:
For testing local, you purpose to use `process.env.FORCE_SENTRY`?

Reply:
jawohl